accounting/entries.go

```diff
@@ -241,3 +241,78 @@ func invoiceEntry(invoice *lnrpc.Invoice, circularReceipt bool,
 		hash, preimage, note, false, convert,
 	)
 }
+
+// paymentReference produces a unique reference for a payment. Since payment
+// hash is not guaranteed to be unique, we use the payments unique sequence
+// number and its hash.
+func paymentReference(sequenceNumber uint64, paymentHash string) string {
+	return fmt.Sprintf("%v:%v", sequenceNumber, paymentHash)
+}
+
+// paymentNote creates a note for payments from our node.
+func paymentNote(preimage string) string {
+	return fmt.Sprintf("Preimage: %v", preimage)
+}
+
+// paymentFeeNote creates a note for a payment fee entry.
+func paymentFeeNote(htlcs []*lnrpc.HTLCAttempt) string {
```

Comment:
Would it make sense to pass in the `payment` itself into these functions so they can select the fields that are relevant for the task at hand?

Reply:
I think it makes sense for when we're passing a lot of params, but going to leave as is for these ones. It's a bit easier for testing to be able to pass expected values in.